python_modules/dagster/dagster/_core/definitions/schedule_definition.py

```diff
@@ -196,15 +197,7 @@ def __init__(
         self._scheduled_execution_time = check.opt_inst_param(
             scheduled_execution_time, "scheduled_execution_time", datetime
         )
-        self._log_key = (
-            [
-                repository_name,
-                schedule_name,
-                scheduled_execution_time.strftime("%Y%m%d_%H%M%S"),
-            ]
```

Comment:
We're changing the scheme here. Instead of timestamp, we're using the tick id. [1]